clients/ethereumjs/Dockerfile

```diff
@@ -1,23 +1,31 @@
+### Build Ethereumjs From Git:
+
 FROM node:16-alpine as build
-RUN apk update && apk add --no-cache bash git jq curl python3 gcc make g++ && rm -rf /var/cache/apk/*
-RUN git clone --depth 1 -b develop-v7 https://github.com/ethereumjs/ethereumjs-monorepo.git && \
-    (cd ethereumjs-monorepo && npm i)
 
+ARG user=ethereumjs
+ARG repo=ethereumjs-monorepo
+ARG branch=develop-v7
+
+RUN echo "Cloning: $user/$repo - $branch" \ 
+    && apk update && apk add --no-cache bash git jq curl python3 gcc make g++ \
+    && rm -rf /var/cache/apk/* \
+    && git clone --depth 1 -b $branch https://github.com/$user/$repo \
```

Comment:
its better to just define an arg REPO_URL (and set it to default) rather than user/repo args, reduces confusion about what user is being refered to